cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala

```diff
@@ -33,13 +33,13 @@ CREATE OR REPLACE DATABASE myDatabase
 
 (★) Create a database named `myDatabase`. If a database with that name exists, then the existing database is deleted and a new one created.
 
-###assertion=update-one
+###assertion=update-two
 //
 
-CREATE DATABASE myDatabase TOPOLOGY 1 PRIMARY 0 SECONDARIES
+CREATE OR REPLACE DATABASE myDatabase TOPOLOGY 1 PRIMARY 0 SECONDARIES
```

Comment:
As the one above already have `CREATE OR REPLACE` it might be nicer to have some variation and go with just `CREATE` (and a new name) or `CREATE IF NOT EXISTS`